.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: '[BUG]'
+labels: 'kind/bug'
+assignees: ''
+
+---
+
+### Rancher Server Setup
+- Rancher version:
+- Installation option (Docker install/Helm Chart):
+   - If Helm Chart, Kubernetes Cluster and version (RKE1, RKE2, k3s, EKS, etc):
+- Proxy/Cert Details:
+
+### Information about the Cluster
+- Kubernetes version:
+- Cluster Type (Local/Downstream):
+   - If downstream, what type of cluster? (Custom/Imported or specify provider for Hosted/Infrastructure Provider):
+<!--
+* Custom = Running a docker command on a node
+* Imported = Running kubectl apply onto an existing k8s cluster
+* Hosted = EKS, GKE, AKS, etc
+ * Infrastructure Provider = Rancher provisioning the nodes using different node drivers (e.g. AWS, Digital Ocean, etc)
+-->
+
+### User Information
+- What is the role of the user logged in? (Admin/Cluster Owner/Cluster Member/Project Owner/Project Member/Custom)
+  - If custom, define the set of permissions:
```

Comment:
It might be useful to know what version of the provider they're using

Reply:
Good point, I totally overlooked it 😅 
Will update.

Reply:
@HarrisonWAffel I added it under the `### Provider Information` header along with Terraform version, could you please take another look? 🙏